freshmaker/lightblue.py

```diff
@@ -1576,8 +1576,10 @@ def _get_images_to_rebuild(image):
         to_rebuild = self._deduplicate_images_to_rebuild(to_rebuild)
         # Get all the directly affected images so that any parents that are not marked as
         # directly affected can be set in _images_to_rebuild_to_batches
+        # And in addition mark all EUS images as directly affected so botas could
+        # correctly file an advisory
         directly_affected_nvrs = {
-            image.nvr for image in images if image.get("directly_affected")
+            image.nvr for image in images if image.get("directly_affected") or is_eus_image(image)
```

Comment:
You need to ensure that the EUS image being rebuilt is the latest image tagged with an autorebuild tag.

For example, if I have child image A built on `rhel-els-container-8.2-4` but the latest is actually `rhel-els-container-8.2-5`, then `rhel-els-container-8.2-4` shouldn't be marked as directly affected.

Reply:
So when we have some EUS image for example `rhel-els-container-8.2-4` that is a parent of the image attached to the advisory, we should check if it's the latest image, and if it's not, rebuild latests image instead of actual parent? Does this make sense?

Reply:
Probably you should update `_filter_out_already_fixed_published_images` to also check the latest EUS images, if the latest EUS image already has the CVE fix addressed, the latest EUS image should be used as parent, and in such cases, we don't need to mark the EUS image as affected.